sklearn/covariance/_shrunk_covariance.py

```diff
@@ -543,8 +543,8 @@ class OAS(EmpiricalCovariance):
     ...                      [.3, .4]])
     >>> rng = np.random.RandomState(0)
     >>> X = rng.multivariate_normal(mean=[0, 0],
-    ...                                    cov=real_cov,
-    ...                                    size=500)
+    ...                                   cov=real_cov,
+    ...                                   size=500)
```

Comment:
I corrected the alignment here, too.

Reply:
But the correct alignment here is for `cov` to be aligned with `mean`, not the bracket, since it's not a part of the list represented by the brackets.

Reply:
Oh, true! I addressed it.